interface_frontend.py

```diff
@@ -188,6 +190,22 @@ def draw_interface(interface_state, window):
                 ):
             draw_card(coordinate, card)
 
+    if interface_state.players:
+        # Other robots background
+        for i in range(len(interface_state.players)):
+            players_background.x = 50 + i * 98
+            players_background.y = 50
+            players_background.draw()
+
+        # Other robots
+        for i, robot in enumerate(interface_state.players):
+            if robot.name in loaded_robots_images.keys():
```

Comment:
```suggestion
            if robot.name in loaded_robots_images:
```

Reply:
Můžu pro vysvětlení? :-D Takhle to prohledá jak klíče tak i hodnoty?